src/xrpld/overlay/detail/PeerImp.cpp

```diff
@@ -1351,8 +1351,8 @@ PeerImp::handleTransaction(
     {
         // If we've never been in synch, there's nothing we can do
         // with a transaction
-        JLOG(p_journal_.debug()) << "Ignoring incoming transaction: "
-                                 << "Need network ledger";
+        JLOG(p_journal_.debug())
+            << "Ignoring incoming transaction: " << "Need network ledger";
```

Comment:
This can be combined into one string.

```suggestion
        JLOG(p_journal_.debug())
            << "Ignoring incoming transaction: Need network ledger";
```